game/src/common/waypoints.rs

```diff
@@ -96,18 +110,19 @@ impl InputWaypoints {
 
             delete_buttons.push(
                 ctx.style()
-                    .btn_plain_destructive
-                    .text("X")
+                    .btn_close()
+                    .override_style(&ctx.style().btn_plain_destructive)
```

Comment:
usually we build the button *from* the style... but some of these "special buttons" like the close button make that a bit awkward, so I added this override style method.

The alternative would be to do something like:

ctx.style().btn_plain_destructive.btn_image(widgetry::include_image_bytes("../../some/hardcoded/path/into/an/external/crate"))